app/presenters/publishing_api_presenters/publication.rb

```diff
@@ -0,0 +1,69 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::Publication < PublishingApiPresenters::Edition
+  include PublishingApiPresenters::WithdrawingHelper
+  include PublishingApiPresenters::PoliticalHelper
+  include PublishingApiPresenters::ApplicabilityHelper
+
+  def links
+    extract_links([
+      :lead_organisations,
+      :organisations,
+      :document_collections,
+      :supporting_organisations,
+      :world_locations
+    ]).merge(
+      ministers: ministers,
+      related_statistical_data_sets: item.statistical_data_set_ids,
+      topical_events: topical_events
+    )
+  end
+
+private
+
+  def schema_name
+    "publication"
+  end
+
+  def details
+    super.merge({
+      body: body,
+      documents: documents,
+      first_public_at: first_public_at,
+      change_history: item.change_history.as_json,
+      emphasised_organisations: item.lead_organisations.map(&:content_id)
```

Comment:
Emphasised organisations are used for search purposes. They include lead organisations, but can include more organisations too. Don't know much more but there is more context in #2585.